src/ontology/cl-edit.owl

```diff
@@ -15507,7 +15507,9 @@ AnnotationAssertion(oboInOwl:creation_date obo:CL_0002205 "2010-09-02T02:38:34Z"
 AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0002205 "CL:1000386"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0002205 "FMA:263220"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0002205 "brush cell of lobular bronchiole"^^xsd:string)
-EquivalentClasses(obo:CL_0002205 ObjectIntersectionOf(obo:CL_0002075 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051)))
+EquivalentClasses(obo:CL_0002205 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002051)))
```

Comment:
Based on the label 'brush cell of lobular bronchiole', this EA may be too broad.

The label suggests this cell type is part of the epithelium of a 'lobular bronchiole'.
The EA asserts this cell type is part of 'epithelium of bronchiole', which suggests other distal structures such as "terminal bronchiole" are included.

This seems compounded by UBERON stating 'epithelium of bronchiole' has exact synonym 'epithelium of lobular bronchiole", which seems to stem from FMA2 and FMA3.